CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ _This release is scheduled to be released on 2023-10-01._
 - Update engine node >=18. v16 reached it's end of life. (#3170)
 - Update typescript definition for modules
 - Cleaned up nunjuck templates
+- Replace `node-fetch` with internal fetch (#2649) and remove `digest-fetch`
```

Comment:
I'd mention it in a BREAKING CHANGE section with "Removed digest emthod form calendar becuase it didnt wokr for ... month)" or similar. Make it more clear. Also maybe post it in the forum to ask if anybody complains?

Reply:
added an extra entry under `Removed`.

> Also maybe post it in the forum to ask if anybody complains?

No, thats waste of time but if you want to do it ...

To be more precise:
- the bug using `digest-fetch` was introduced with release `2.15.0` and nobody complained about it until today (I follow the forum and discord and github). So this is a "good long running test" thats it is not used ...
- I don't know how often calendar auth is used in general, these are edge cases. If you use a public provider as e.g. google you get a private url. I don't know if public providers offer digest auth.
- If really someone is using digest auth (with a very old release) he can also easily transfer to basic auth. Using basic auth with https is more secure as digest auth reading [this article](https://en.wikipedia.org/wiki/Digest_access_authentication)

Reply:
Awesome work! Thx a lot!!!